packages/runtime/container-runtime/src/garbageCollection.ts

```diff
@@ -511,7 +518,18 @@ export class GarbageCollector implements IGarbageCollector {
             // other existing documents, GC is enabled.
             this.gcEnabled = prevSummaryGCVersion > 0;
             this.sweepEnabled = metadata?.sweepEnabled ?? false;
+            // These three numbers add up to the Sweep Timeout
             this.sessionExpiryTimeoutMs = metadata?.sessionExpiryTimeoutMs;
+            this.maxSnapshotCacheDurationMs = metadata?.maxSnapshotCacheDurationMs;
```

Comment:
I'm concerned here... for existing files where the value has been hardcoded, it will be missing from metadata but should default to 5 days not undefined.

How can we distinguish this from the case where the driver didn't implement this policy??

Reply:
I think we can solve this.

Just always write a value going forward, either a number, or "infinity" or some other special value (or -1?) indicating the driver didn't implement the policy in first session.

So then undefined means backfill with 5 days default.